inspirehep/modules/theme/ext.py

```diff
@@ -89,3 +92,12 @@ def setup_app(self, app):
                             force_lower=False)
         del gravatar
         return app
+
+    def init_weblinks_dictionary(self):
+
+        weblinks_dictionary = {}
+        with open(os.path.join('inspirehep', 'weblinks.kb'), 'r') as file:
```

Comment:
This will not work correctly as it generates the relative path `inspirehep/weblinks.kb`. So running this on a webserver or anywhere other than from `$(VIRTUAL_ENV)/src/inspirehep` will fail. To get the right path reliably, take a look at this approach: https://pythonhosted.org/setuptools/pkg_resources.html#resource-extraction

```pkg_resources.resource_filename('inspirehep', 'weblinks.kb')```